starknet_py/abi/v2/model.py

```diff
@@ -83,7 +83,7 @@ class Impl:
         Constructor
     ]  #: Contract's constructor. It is None if class doesn't define one.
     l1_handler: Optional[
```

Comment:
I would prefer to rename it to `l1_handlers` but this is a bit problematic given that in some places we use `Union[AbiV0, AbiV2]`, and `AbiV0` contains `l1_handler`, so I decided to retain the same name here.